src/nikhilMJun20/AllDiffIndex.java

```diff
@@ -0,0 +1,25 @@
+/*Find all differ index from given arrays.
+i/p:- arr1: {10,2,9,14,3}
+arr2: {10,2,18,14,13}
+o/p:- Values are not matching at index -> 2
+Values are not matching at index -> 4
+*/
+
+package nikhilMJun20;
+
+public class AllDiffIndex {
+	void displayAllDiffIndex(int[] arr1, int[] arr2) {
+		for(int index=0; index<arr1.length; index++) {
```

Comment:
Nikhil, to write directly for loop is wrong. Because first, you should check that Both arrays length is equal or not. 
If length is not equal then no need to proceed further. 

Please add this change to all your methods.

Reply:
When do you write like this " index<arr1.length; "  interview directly ask you that on which basis you decided that length should be as arr1.length why not arr12.length ? 
So initially if you write if (arr1.length == arr2.length) this then you can iterate loop till any arrays length.

Reply:
Hi @KrishnaKTechnocredits,
Made this change to all the array programs.
Thanks!